README.md

```diff
@@ -7,7 +7,7 @@ This branch contains test data to be used for automated testing with the [nf-cor
 | File	                | Description	                                                                                              |
 |-----------------------|-----------------------------------------------------------------------------------------------------------|
 | `samplesheet.csv`     | Sample information file                                                                                   |
-| `fast5/`              | subset of fast5 files from direct cDNA Nanopore reads for MCF7 (Breast Cancer) cell line                  |
+| `fast5/`              | Subset of fast5 files from direct cDNA Nanopore reads for HepG2 (Liver Cancer) and K562 (Leukemia) cell lines                  |
```

Comment:
Add a listing of both directories in `fast5`  i.e. `fast5/barcoded/` and `fast5/nonbarcoded/`

Reply:
Ive changed this now but Im still unsure which cell lines the `fast5` data from the `barcoded` and `non-barcoded` data come from. Are they from the same samples? Maybe add a more relevant description for each in the `Files` section. Also, could rename the sample in `samplesheet_nonbarcoded.csv` to something a bit more accurate. Ive put in `K562_RUN1_REP3` for now as a placeholder.